src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/DisposeObjectsBeforeLosingScopeTests.cs

```diff
@@ -1957,9 +1957,7 @@ void M1(A a)
 }
 ",
             // Test0.cs(16,20): warning CA2000: In method 'void Test.M1(A a)', call System.IDisposable.Dispose on object created by 'new A()' before all references to it are out of scope.
-            GetCSharpResultAt(16, 20, "void Test.M1(A a)", "new A()"),
-            // Test0.cs(20,20): warning CA2000: In method 'void Test.M1(A a)', call System.IDisposable.Dispose on object created by 'new A()' before all references to it are out of scope.
```

Comment:
This was a false positive - my bad to not have caught this while adding the unit test.